### PR TITLE
Add :created_at attribute to UPP serializer

### PR DESCRIPTION
**Pull request by srallen**
This adds `:created_at` attribute to the user project preferences serializer. The first a/b split that Gravity Spy ran was to study behavior with users new to the project and the only way we can determine that is by the existence of the user project preferences for the project. If they want to run any future experiments with new users, it will be useful to have the `created_at` timestamp available via the API and possibly log it to geordi for the researcher's use in data analysis.

I looked through the spec files and didn't see anything that I would need to update in the tests, but do let me know if I missed something. 

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by srallen** [comment]
Cool, I'll take a look at the travis failure. Definitely missed something.

**Comment by marten** [comment]
To me that failure seemed like a random failure that you didn't cause. This one looks like the test didn't take into account that Time.now can tick over to the next second while a test runs (it was expecting a timestamp like 12:34:56 but got 12:34:57). But again, I think it's not your fault. I told Travis to go retry the test suite, we'll see if it magically passes next time.

File: app/serializers/user_project_preference_serializer.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ class UserProjectPreferenceSerializer
   include CachedSerializer
 
   attributes :id, :email_communication, :preferences, :href,
-    :activity_count, :activity_count_by_workflow, :settings
+    :activity_count, :activity_count_by_workflow, :settings,
+    :created_at
```
Comment (marten) [inline app/serializers/user_project_preference_serializer.rb]: Might want to add `:updated_at` while you're at it.

**Comment by srallen** [comment]
Hm, good to know about the test. I only briefly looked over the logs before I left yesterday.

**Pull request merged**